polyply/tests/test_data/library_tests/martini3/PPE/polyply/PPE.itp

```diff
@@ -0,0 +1,442 @@
+36    34    33    35    2  0.0        50.0       
+36    38    39    37    2  0.0        50.0       
+40    38    37    39    2  0.0        50.0       
+40    42    43    41    2  0.0        50.0       
+44    42    41    43    2  0.0        50.0       
+44    46    47    45    2  0.0        50.0       
+48    46    45    47    2  0.0        50.0       
+48    50    51    49    2  0.0        50.0       
+52    50    49    51    2  0.0        50.0       
+52    54    55    53    2  0.0        50.0       
+56    54    53    55    2  0.0        50.0       
+56    58    59    57    2  0.0        50.0       
+60    58    57    59    2  0.0        50.0       
+60    62    63    61    2  0.0        50.0       
+64    62    61    63    2  0.0        50.0       
+64    66    67    65    2  0.0        50.0       
+68    66    65    67    2  0.0        50.0       
+68    70    71    69    2  0.0        50.0       
+72    70    69    71    2  0.0        50.0       
+72    74    75    73    2  0.0        50.0       
+76    74    73    75    2  0.0        50.0       
+76    78    79    77    2  0.0        50.0       
+80    78    77    79    2  0.0        50.0       
+80    82    83    81    2  0.0        50.0       
+84    82    81    83    2  0.0        50.0       
+84    86    87    85    2  0.0        50.0       
+[     constraints      ]
+1     2     1 0.32500 
+1     3     1 0.32500 
+2     3     1 0.32500 
+5     6     1 0.32500 
+5     7     1 0.32500 
+6     7     1 0.32500 
+9     10    1 0.32500 
+9     11    1 0.32500 
+10    11    1 0.32500 
+13    14    1 0.32500 
+13    15    1 0.32500 
+14    15    1 0.32500 
+17    18    1 0.32500 
+17    19    1 0.32500 
+18    19    1 0.32500 
+21    22    1 0.32500 
+21    23    1 0.32500 
+22    23    1 0.32500 
+25    26    1 0.32500 
+25    27    1 0.32500 
+26    27    1 0.32500 
+29    30    1 0.32500 
+29    31    1 0.32500 
+30    31    1 0.32500 
+33    34    1 0.32500 
+33    35    1 0.32500 
+34    35    1 0.32500 
+37    38    1 0.32500 
+37    39    1 0.32500 
+38    39    1 0.32500 
+41    42    1 0.32500 
+41    43    1 0.32500 
+42    43    1 0.32500 
+45    46    1 0.32500 
+45    47    1 0.32500 
+46    47    1 0.32500 
+49    50    1 0.32500 
+49    51    1 0.32500 
+50    51    1 0.32500 
+53    54    1 0.32500 
+53    55    1 0.32500 
+54    55    1 0.32500 
+57    58    1 0.32500 
+57    59    1 0.32500 
+58    59    1 0.32500 
+61    62    1 0.32500 
+61    63    1 0.32500 
+62    63    1 0.32500 
+65    66    1 0.32500 
+65    67    1 0.32500 
+66    67    1 0.32500 
+69    70    1 0.32500 
+69    71    1 0.32500 
+70    71    1 0.32500 
+73    74    1 0.32500 
+73    75    1 0.32500 
+74    75    1 0.32500 
+77    78    1 0.32500 
+77    79    1 0.32500 
+78    79    1 0.32500 
+81    82    1 0.32500 
+81    83    1 0.32500 
+82    83    1 0.32500 
+85    86    1 0.32500 
+85    87    1 0.32500 
+86    87    1 0.32500 
+#ifdef POSRES 
```

Comment:
For the automatic check, I think we cannot deal with position_restraints that have this force constant syntax. Hence the fail of the test. Can you remove this section that deals with the positions_restraints. 

Also are they needed for equilibriation etc.? If so I think we could also implement them in polyply by adding position restraints to the ff-file blocks. However, then you cannot add the #define directive automatically. Not sure that would help then.

Reply:
check